packages/@o3r/core/schematics/rule-factories/store/index.ts

```diff
@@ -131,7 +131,7 @@ const rootReducers = {
   ${isApplicationThatUsesRouterModule(tree) ? 'router: routerReducer' : ''}
 };
 
-const metaReducers = [storageSync.localStorageSync()];
+const metaReducers = [storageSync.localStorageSync];
```

Comment:
You lose the context of the class by doing this

```suggestion
const metaReducers = [(reducer) => storageSync.localStorageSync(reducer)];
```
or
```suggestion
const metaReducers = [storageSync.localStorageSync.bind(storageSync)];
```

Reply:
first one does not work as well, only the bind solution does work. I will go back with my original fix which should not impact the applications